apps/console/src/features/claims/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -91,10 +92,10 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
     const [ hasMapping, setHasMapping ] = useState<boolean>(false);
     const [ mappingChecked, setMappingChecked ] = useState<boolean>(false);
 
-    const nameField = useRef<HTMLElement>(null);
-    const regExField = useRef<HTMLElement>(null);
-    const displayOrderField = useRef<HTMLElement>(null);
-    const descriptionField = useRef<HTMLElement>(null);
+    const nameField: React.MutableRefObject<HTMLElement> = useRef<HTMLElement>(null);
```

Comment:
Let's explicitly import `MutableRefObject` from "react".
```suggestion
    const nameField: MutableRefObject<HTMLElement> = useRef<HTMLElement>(null);
```